internal/clients/readiness.go

```diff
@@ -105,7 +105,7 @@ func (c DefaultReadinessChecker) checkPendingClient(
 	client, err := c.factory.CreateAdminAPIClient(ctx, pendingClient)
 	if err != nil {
 		// Despite the error reason we still want to keep the client in the pending list to retry later.
-		c.logger.V(util.DebugLevel).Info("pending client is not ready yet",
+		c.logger.V(util.DebugLevel).Info("Pending client is not ready yet",
```

Comment:
Not all logs in this file are updated:
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/clients/readiness.go#L98
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/clients/readiness.go#L129
Looks like log messages in `fmt.Sprintf` or in different line with `logger.Info`/`logger.Error` are not touched.

Reply:
@randmonkey  Yes, thank you for pointing it out.
For the convenience of review, I have split them up.
So not all modifications will be visible in the current PR.

When all PRs are merged, the relevant parts will be fully modified.

Reply:
I think this will not block the review and merging of this PR.

Reply:
So what is your standard to split PRs? If you want split PRs by modified files, I think it would be better to finish all occurrences in a file at one time.

Reply:
@randmonkey I have updated the remaining parts that need to be updated in these files. PTAL, thanks